### PR TITLE
Various improvements of the German translation

### PR DESCRIPTION
**Pull request by sumpfralle**
The following changes are included:
* properly handle "Präfix" as neuter
* add missing hyphens to many appearances of "Präfix"
* fix spelling
* add few missing translations

### CODE REVIEW

File: modules/luci-base/po/de/base.po [hunk 1]
```diff
@@ -136,7 +136,7 @@ msgstr "802.11w: Schutz von Management-Frames aktivieren"
 
 #: modules/luci-mod-network/luasrc/model/cbi/admin_network/wifi.lua:1130
 msgid "802.11w maximum timeout"
-msgstr "802.11w: Maximales Timeout"
+msgstr "802.11w: Maximale Zeitbegrenzung"
```
Comment (feckert) [inline modules/luci-base/po/de/base.po]: That sounds weird in german is there not a nice translation? I think timeout is ok or what do you think about `Zeitüberschreitung`?

File: modules/luci-base/po/de/base.po [hunk 2]
```diff
@@ -1237,7 +1237,7 @@ msgstr ""
 
 #: modules/luci-mod-system/luasrc/model/cbi/admin_system/leds.lua:52
 msgid "Custom flash intervall (%s)"
-msgstr ""
+msgstr "Angepasste Blink-Periode"
```
Comment (feckert) [inline modules/luci-base/po/de/base.po]: The '(%s)` is missing and hhat sounds also weird in german is there not a nicer translation?

File: modules/luci-base/po/de/base.po (hunk 1 above)
Comment (sumpfralle) [reply modules/luci-base/po/de/base.po]: `Zeitüberschreitung` does not feel like the correct translation of `timeout` to me. Other people may have a better feeling for this ...

File: modules/luci-base/po/de/base.po (hunk 2 above)
Comment (sumpfralle) [reply modules/luci-base/po/de/base.po]: Indeed, I forgot the formatting. And I do not have really better wording at the moment.

**Comment by sumpfralle** [comment]
Thank you for reviewing my changes!

Since I am not focused on translations, I reduced my drive-by-commit to the non-controversial set of changes.

Happy hacking!

**Comment by feckert** [comment]
Please add `Signed-off-by: xxx` to your commit message

**Comment by sumpfralle** [comment]
Thank you for the hint - done!

**Comment by feckert** [comment]
Sorry but you must also update the commit head to 

> luci-base Update German translation 

If this is done I think we are ready to merge

**Comment by sumpfralle** [comment]
no problem - done ...

Thank you for maintaining luci!

**Comment by feckert** [comment]
Thanks for your contribution.

**Pull request merged**